src/script/main/SingleInstanceHandler.js

```diff
@@ -90,9 +91,8 @@ z.main.SingleInstanceHandler = (() => {
      * Does not check for the id of the running instance and thus cannot be
      * invoked once the registering of the current instance has been done.
      *
-     * @param {Function} listener - A listener to be removed.
      * @throws {Error} - When the current app has already been registered.
-     * @returns {void} - Returns nothing.
+     * @returns {boolean} - Returns nothing.
```

Comment:
Should return something now :slightly_smiling_face:

Reply:
Why "now"? It has always returned something. Just the JSdoc annotation has been wrong.